container/worker/run_openqa_worker.sh

```diff
@@ -1,18 +1,25 @@
 #!/bin/bash
-# shellcheck disable=SC2012
+# shellcheck disable=SC2012,SC2154
 set -e
+set -x
```

Comment:
this looks like temporary debugging code

Reply:
Removed